src/client/utils/style.js

```diff
@@ -17,7 +17,7 @@ export function isStyle (instance) {
     if (instance instanceof nativeMethods.styleClass)
         return true;
 
-    if (instance && typeof instance === 'object' && instance.border !== void 0) {
+    if (instance && typeof instance === 'object' && instance.border !== void 0 && typeof instance.toString === 'function') {
```

Comment:
`String({ border: 0, toString: null}) `

throws 

`Uncaught TypeError: Cannot convert object to primitive value`